### DISCUSSION TITLE
Defect Dojo findings import issue.

### DISCUSSION THREAD

**Discussion by Noonster77** [DefectDojo/django-DefectDojo#11400] [Q&A]
I have a .csv file that is a direct export from tenable.sc in the plugin name it shows a finding like "Putty < 0.81". I have opened up the sheet and it does show the < symbol. When I import the scan as a tenable scan it does import the findings, but the title says "Putty &lt" instead of the < symbol. Any ideas? Thanks!

**Comment by manuel-sommer** [comment]


**Reply by Noonster77** [reply]
Thanks opened up an issue and provided sample files.

**Reply by manuel-sommer** [reply]
I guess this can be closed because of the issue @mtesauro . Maybe also switch the category.
